guardrails/telemetry/runner_tracing.py

```diff
@@ -83,6 +85,9 @@ def trace_step_wrapper(*args, **kwargs) -> Iteration:
                 name="step",  # type: ignore
                 context=current_otel_context,  # type: ignore
             ) as step_span:
+                step_span.set_attribute(
+                    SpanAttributes.OPENINFERENCE_SPAN_KIND, "GUARDRAIL"
+                )
```

Comment:
Second half of the comment at the top of this file concerning optional dependencies.
```suggestion
                if SpanAttributes is not None:
                    step_span.set_attribute(
                        SpanAttributes.OPENINFERENCE_SPAN_KIND, "GUARDRAIL"
                    )
```